app/javascript/components/menus/edit/edit-menu-container.vue

```diff
@@ -0,0 +1,259 @@
+                  {{ recipeSelected.name }}
+                </selected-recipes-card>
+              </div>
+            </div>
+            <div
+              class="flex h-6 bg-gray-50 font-sans font-light text-base text-black self-stretch mb-3"
+              v-else
+            >
+              {{ $t('msg.menus.noRecipes') }}
+            </div>
+            <div class="flex justify-end py-4 px-2 bg-gray-200 border border-gray-300 box-border self-stretch">
+              <div class="w-auto h-6 font-bold text-base text-black mx-2">
+                {{ $t('msg.menus.menuPrice') }} {{ totalMenuPrice | currency }}
+              </div>
+            </div>
+          </div>
+          <button
+            class="bg-green-500 hover:bg-green-700 text-white mx-2 my-2 h-10 font-bold py-2 px-6 rounded shadow-md flex-shrink-0 focus:outline-none"
+            @click="editMenu"
+          >
+            {{ $t('msg.menus.saveChanges') }}
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { getRecipes } from '../../../api/recipes.js';
+import { getMenu, updateMenu } from '../../../api/menus.js';
+import SelectedRecipesCard from '../base/selected-recipes-card.vue';
+import AddRecipeCard from '../base/add-recipe-card';
+import { getPriceOfSelectedIngredient } from '../../../utils/recipeUtils';
+
+export default {
+  props: {
+    menuId: { type: Number, required: true },
+  },
+  components: {
+    SelectedRecipesCard,
+    AddRecipeCard,
+  },
+
+  data() {
+    return {
+      initialRecipes: [],
+      recipes: [],
+      error: '',
+      query: '',
+      menuName: '',
+      menuPortions: 0,
+      selectedRecipes: [],
+      deletedRecipes: [],
+    };
+  },
+  computed: {
+    totalMenuPrice() {
+      const recipesPrices = this.selectedRecipes.map(element => ((
+        element.recipeIngredients.data.reduce((recipePrice, recipeIngredient) =>
+          recipePrice + this.getPriceOfSelectedIngredient(recipeIngredient.attributes), 0)
+      ) * element.quantity),
+      );
+
+      return recipesPrices.reduce((menuPrice, recipePrice) =>
+        menuPrice + recipePrice, 0);
+    },
+  },
+  async created() {
+    try {
+      const menuResponse = await getMenu(this.menuId);
+      const response = await getRecipes();
+      this.recipes = response.data.data.map((element) => ({
+        id: element.id,
+        ...element.attributes,
+      }));
+      this.useMenuInfo(menuResponse.data.data);
+      this.error = '';
+    } catch (error) {
+      this.error = error;
+    }
+  },
+  methods: {
+    async editMenu() {
+      try {
+        const updatedMenu = this.getUpdatedMenu();
+        await updateMenu(this.menuId, updatedMenu);
+        window.location = '/menus';
+      } catch (error) {
+        this.error = error;
+      }
+    },
+    useMenuInfo(menu) {
+      this.menuName = menu.attributes.name;
+      this.portions = menu.attributes.portions;
+      this.portions = menu.id;
```

Comment:
Acá se está modificando dos veces `this.portions`, me imagino que el segundo está mal o no?